src/SourcemapToolkit.SourcemapParser/SourceMapParser.cs

```diff
@@ -36,7 +38,15 @@ public SourceMap ParseSourceMap(StreamReader sourceMapStream)
 				RemoveExtraSpaceFromList(parsedMappings);
 				RemoveExtraSpaceFromList(deserializedSourceMap.Sources);
 				RemoveExtraSpaceFromList(deserializedSourceMap.Names);
-				RemoveExtraSpaceFromList(deserializedSourceMap.SourcesContent);
+
+				if (_removeSourcesContent && deserializedSourceMap.SourcesContent != null)
+				{
```

Comment:
So, we read the sourcesContent and then clean it. 
Is this by design? Can't we just not read it? What are we trying to solve here?

Reply:
this part is parsing the map file when we load it
for the same reason we are doing RemoveExtraSpaceFromList here (to save memory) I added the ability to clean SourcesContent which is not required in most cases for deobfuscation